vignettes/Life_Data_Analysis_Part_I.Rmd

```diff
@@ -21,28 +23,48 @@ knitr::opts_chunk$set(
+then the probability that a unit has failed at _t_ is defined by its 
+_CDF (cumulative distribution function)_ _F(t)_.
 
 $$ P(T\leq t) = F(t) $$
 
-In order to obtain an estimate of the cumulative failure probability for each observation $t_1, t_2, ..., t_n$ two approaches are possible. Using a parametric lifetime distribution requires that the underlying assumptions for the sample data are valid. If the distribution-specific assumptions are correct, the model parameters can be estimated and the CDF is computable. But if the required conditions could not be met, interpretations and derived conclusions are not reliable. A more general approach for the calculation of the cumulative failure probability is to use non-parametric statistical estimators $\hat{F}(t_1), \hat{F}(t_2), ..., \hat{F}(t_n)$. In comparison to a parametric distribution no general assumptions must be held. For non-parametric estimators, an ordered sample of size n is needed. Starting at 1, the ranks $i \in \{1, 2, ..., n \}$ are assigned to the ascending sorted sample values. Since there is a known relationship between ranks and corresponding ranking probabilities a CDF can be calculated.  
-
-But rank distributions are systematically skewed distributions and thus the median value instead of the expected value $E\left[F\left(t_i\right)\right] = \frac{i}{n + 1}$ is used for estimation [^note1]. This skewness is visualized in _Figure 1_. 
+In order to obtain an estimate of the _CDF_ for each observation $t_1, t_2, ..., t_n$ 
+two approaches are possible. Using a parametric lifetime distribution requires that 
+the underlying assumptions for the sample data are valid. If the distribution-specific 
+assumptions are correct, the model parameters can be estimated and the _CDF_ is 
+computable. But if conditions could not be met, interpretations and derived conclusions 
+are not reliable.
+
+A more general approach for the calculation of cumulative failure probabilities is 
+to use non-parametric statistical estimators $\hat{F}(t_1), \hat{F}(t_2), ..., \hat{F}(t_n)$. 
+In comparison to a parametric distribution no general assumptions must be held. 
+For non-parametric estimators, an ordered sample of size $n$ is needed. Starting 
+at $1$, the ranks $i \in \{1, 2, ..., n \}$ are assigned to the ascending sorted 
+sample values. Since there is a known relationship between ranks and corresponding 
+ranking probabilities a _CDF_ can be determined.  
+
+But rank distributions are systematically skewed distributions and thus the median 
+value instead of the expected value $E\left[F\left(t_i\right)\right] = \frac{i}{n + 1}$ 
+is used for the estimation [^note1]. This skewness is visualized in _Figure 1_. 
 
 [^note1]: Kapur, K. C.; Lamberson, L. R.: _Reliability in Engineering Design_, 
           _New York: Wiley_, 1977, pp. 297-301  
 
 ```{r rank densities, fig.cap = "Figure 1: Densities for different ranks i in samples of size n = 10.", message = FALSE}
```

Comment:
Chunk titles with whitespace may create non-portable file names (e.g. plots in the figures folder)